core/app/services/spree/locales/set_fallback_locale_for_store.rb

```diff
@@ -0,0 +1,17 @@
+module Spree
+  module Locales
+    class SetFallbackLocaleForStore
+      def call(store:)
+        store_default_locale = store.default_locale
+        fallbacks = store.supported_locales_list.each_with_object({}) do |locale, object|
+          object[locale] = [store_default_locale]
+        end
```

Comment:
You can also consider doing:
```suggestion
        fallbacks = store.supported_locales_list.to_h { |locale| [locale, [store_default_locale]] }
```

Reply:
Hm, isn't it more explicit and clear when we do `each_with_object`?

Reply:
I don't have a strong preference for either to be honest. `each_with_object` probably looks more familiar to most readers. In defense of `#to_h` - to me it's clear in the way it just deals with simple `[key, value]` objects instead of having this "state" object that's initialised in a loop. I like the original as well though, hence my original comment mentioning the suggestion as something to consider.